app/src/main/java/org/jellyfin/androidtv/util/BackgroundManagerUtils.kt

```diff
@@ -0,0 +1,42 @@
+package org.jellyfin.androidtv.util
+
+import android.app.Activity
+import android.graphics.drawable.Drawable
+import androidx.leanback.app.BackgroundManager
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation
+import com.bumptech.glide.load.resource.bitmap.CenterInside
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+
+/** Created by Oriol on 30/05/20. **/
```

Comment:
We don't add "created by" comments to files. We only have them in some other files that were inherited from the old Emby code.